### DISCUSSION TITLE
Weekly Digest: No cut-off date, empty strings, Paris, etc

### DISCUSSION THREAD

**Discussion by richfab** [MobilityData/gbfs#601] [Announcements]
🗳️ Following the [vote](https://mobilitydata-io.slack.com/archives/CNXA9ASBV/p1706825862045199) (0 vote in favor, 1 vote against), there will be **no cut-off date** to implement the v3.0-RC features at this time.

This gives you, producers and consumers, more time to implement the features that you’d like to see included in v3.0:

- 🅿️ **Geofencing made simple** ([#481](https://github.com/MobilityData/gbfs/pull/481)) - Missing a consumer. Produced by [Flamingo](https://data.rideflamingo.com/gbfs/3/wellington/gbfs.json), [TIER](https://data-sharing.tier-services.io/tier_paris/gbfs/3.0/), [Check](https://api.ridecheck.app/gbfs/v3/almere/gbfs.json) and [ENTUR](https://api.dev.entur.io/mobility/v2/gbfs/v3beta/manifest.json).
- 🕐 **Opening hours format** ([#328](https://github.com/MobilityData/gbfs/pull/328)) - Missing a consumer. Produced by [Flamingo](https://data.rideflamingo.com/gbfs/3/wellington/gbfs.json), [TIER](https://data-sharing.tier-services.io/tier_paris/gbfs/3.0/), [Check](https://api.ridecheck.app/gbfs/v3/almere/gbfs.json))
- ⌛ **Timestamps format** ([#522](https://github.com/MobilityData/gbfs/pull/522)) - Missing a consumer. Produced by [ENTUR](https://api.dev.entur.io/mobility/v2/gbfs/v3beta/manifest.json).
- 💵 **Pricing per mile** ([#546](https://github.com/MobilityData/gbfs/pull/546)) - Missing a producer. Consumed by Transit App.
- 🎟️ **Reservation price** ([#457](https://github.com/MobilityData/gbfs/pull/457)) - Missing a producer and consumer.

🙌 Protecting publishers and riders, reducing implementation costs, and improving the rider experience, are some of the many exciting benefits of GBFS v3.0. [Read more](https://mobilitydata.org/gbfs-v3-0-is-live-heres-why-youll-love-it/).

Issue in discussion: [#600](https://github.com/MobilityData/gbfs/issues/600) - REQUIRED Strings : should the spec say they MUST NOT be empty ("")?

🚲 I was in Paris last weekend and we definitely get a feeling of Amsterdam like city now! Paris bike lane usage **doubled** in one year ([source](https://www.lemonde.fr/les-decodeurs/article/2023/11/11/a-paris-la-frequentation-des-pistes-cyclables-a-double-en-un-an_6199510_4355770.html) in French).

🇫🇷  Speaking of Paris, I will be at the [Autonomy Mobility World Expo](https://www.amwe.world/en) on March 20-21. The topics covered will be shared mobility 2.0, MaaS, data, AI and mobility (program [here](https://www.amwe.world/en/sessions)). Write to me if you go, I’ll be happy to **chat in person**!

https://github.com/MobilityData/gbfs/assets/2423604/6bb07e91-5161-4b56-b7fb-4eeaeffb8f51

**Comment by sven4all** [comment]
As also mentioned in #602 

We use 'Geofencing made simple' as a consumer in the dashboarddeelmobiliteit https://dashboarddeelmobiliteit.nl/.

https://github.com/Stichting-CROW/dd-importer-v2/blob/master/process/service_area.go

It's good to mention that we require our publishers to use the following global rules:

ride_start_allowed = false
ride_end_allowed = false
ride_through_allowed = true

**Reply by richfab** [reply]
Hi @sven4all and welcome 👋 
This makes 'Geofencing made simple' an official feature of v3.0 🎉 
Thanks for implementing and for sharing!
